docs/articles/pod-security-standards.md

```diff
@@ -0,0 +1,176 @@
+# Pod Security Standards
+
+Kubernetes allows one to apply security policies to pods and enforce them with
+[a built-in
+controller](https://kubernetes.io/docs/concepts/security/pod-security-admission/)
+at the namespace level. Defined as [*Pod Security
+Standards*](https://kubernetes.io/docs/concepts/security/pod-security-standards/),
+there are three builtin profiles ranging from most permissive to most
+restrictive.
+
+This guide provides a path to bringing your Testkube installation
+into compliance with the most restrictive profile:
+[`restricted`](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted).
+
+## Updating an Existing Installation
+
+In restricted mode, the pod containers must not run as root so to avoid downtime
+and preserve our old state we should run a few jobs to change ownership of some
+files in the existing persistent volumes.
+
+### Control Plane
+
+In the namespace hosting your Testkube control plane installation `kubectl apply`
+a job which `chown`s files within the persistent volume of the NATS stateful
+set, for example:
+
+```yaml {5-6,14-15,24-25}
+apiVersion: batch/v1
```

Comment:
Does this job need to have some sort of nodeSelectors to be placed on the same node as NATS pods because of accessMode?